src/test/java/com/puppycrawl/tools/checkstyle/api/LocalizedMessageTest.java

```diff
@@ -81,7 +87,9 @@ public void testCountryIsValid() {
      */
     @Test
     public void testLocaleIsSupported() {
-        if (!Locale.ENGLISH.getLanguage().equals(DEFAULT_LOCALE.getLanguage())) {
+        final String language = DEFAULT_LOCALE.getLanguage();
+        if (!Locale.ROOT.getLanguage().equals(language)
```

Comment:
Not sure which is better: `Locale.ROOT.getLanguage()` is actually an empty string, so it can be simple `language.isEmpty()'.

Reply:
I think check for `language.isEmpty()` is better , as will be same as in other tests, so it will look like a family

Reply:
done